### ISSUE TITLE
The param is error in LogFactory.getLog

### ISSUE THREAD

**Issue by tdp100** [aws/aws-sdk-java#392]
[com.huawei.http.RepeatableInputStreamRequestEntity](https://github.com/aws/aws-sdk-java/blob/master/aws-java-sdk-core/src/main/java/com/amazonaws/http/RepeatableInputStreamRequestEntity.java)
```java
    /** Shared logger for more debugging information */
    private static final Log log = LogFactory.getLog(AmazonHttpClient.class);
```
should be:
```java
    /** Shared logger for more debugging information */
    private static final Log log = LogFactory.getLog(RepeatableInputStreamRequestEntity.class);
```

**Comment by hanshuo-aws** [comment]
@tdp100 Looking at the comment above that line, it looks like we intentionally did it, presumably to aggregate more relevant loggings to the same class (AmazonHttpClient).
But I do agree with you that this is somewhat anti-pattern and is hard to maintain in the long run, so I will make this change for our next release.

**Comment by hanshuo-aws** [comment]
Fixed in 1.9.31 version.
